src/main/java/uk/gov/hmcts/darts/common/service/impl/RetrieveCoreObjectServiceImpl.java

```diff
@@ -43,6 +45,7 @@ public class RetrieveCoreObjectServiceImpl implements RetrieveCoreObjectService
     private final DefendantRepository defendantRepository;
     private final ProsecutorRepository prosecutorRepository;
     private final AuthorisationApi authorisationApi;
+    private final CurrentTimeHelper currentTimeHelper;
```

Comment:
This is unused (but I think it should be used, see other comments on use of `CurrentDateTime.now()`